.github/workflows/validate_plugin.yml

```diff
@@ -0,0 +1,71 @@
+name: Plugin Validation
+
+on:
+  pull_request:
+    branches:
+      - "master"
+    paths:
+      - 'community-plugins.json'
+
+jobs:
+  plugin-validation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - run: npm install axios
+      - uses: actions/github-script@v5
+        with:
+          script: |
+            const fs = require('fs');
+            const axios = require('axios');
+            const author = context.payload.sender.login
+            const plugins = JSON.parse(fs.readFileSync('community-plugins.json', 'utf8'));
```

Comment:
Maybe also validate this? I'm not sure if there's a way but sometimes plugin authors don't add their plugin last. I was hoping there'd be a way to tell based on the PR but I don't think that's possible?

Reply:
One could check if the author is the same as the author in the repo field. I think that would be good enough, right?

Reply:
Could add that as a warning - sometimes people do use a different name as their github username. Another thing I had in mind is just validating the JSON which we have a check for as well (here maybe just add a try-catch).

Reply:
What I meant is using the author thats inside the repo string (`<author>/<repo>`). That should be the same as the one opening the PR right?

Reply:
Ohh yes you're right! Most of the time it matches. I've only seen once or twice where one collaborator owns the repo and another collaborator submits the PR, but that should be fine.